package.json

```diff
@@ -29,6 +29,7 @@
     "js-cookie": "^2.1.4",
     "lunr": "^2.3.6",
     "moment": "^2.24.0",
+    "mqtt": "^3.0.0",
```

Comment:
?

Reply:
Some tests failed because mqtt dependency is missing

Reply:
I made the effort and checked: that module is used by the Javascript IoT guide.

1. this change should appear in your PR description
2. the package-lock.json file must also be updated (with an up-to-date version of the npm CLI).
3. since this is only used to test the documentation, then that module must be installed in the `devDependencies` part of the package.json file

Reply:
This should already be fixed without needing to include the module: https://github.com/kuzzleio/documentation/pull/304/files#diff-594d211f65655f937e62cb7d0f75fb99R16

Reply:
So it should be removed from this project's package.json file.